docs/tutorials/edx-platform.rst

```diff
@@ -0,0 +1,124 @@
+.. _edx_platform:
+
+Working on edx-platform as a developer
```

Comment:
This is fantastic!! 🎉 

I do think this guide should be linked to from the [Open edX development guide](https://docs.tutor.overhang.io/dev.html), if not entirely absorbed by it. I also think we ought to avoid confusion by removing the old [XBlock and edx-platform plugin development](https://docs.tutor.overhang.io/dev.html#xblock-and-edx-platform-plugin-development) section.

Furthermore, I recommend de-emphasizing the [Override docker-compose volumes](https://docs.tutor.overhang.io/dev.html#override-docker-compose-volumes) and [Manual bind-mount to any directory](https://docs.tutor.overhang.io/dev.html#override-docker-compose-volumes) sections. Perhaps those will still be useful to some advanced developers, but I think that most developers will only need to know the new instructions you wrote here. I don't want the average developer to get overwhelmed trying to understand the instructions which only now apply to power-users.

Reply:
You are absolutely right. I did the following:

- Add a link to the tutorial at the top of the Open edX development guide.
- Removed the old xblock and edx-platform plugin development section
- Added a warning to the "override docker-compose volumes" section.
- Moved some content to the tutorial.